translations/messages+intl-icu.en_GB.yaml

```diff
@@ -244,7 +244,7 @@ settings.index.title: Settings
 tickets.actors: Actors
 tickets.actors.edit.title: 'Edit the actors'
 tickets.assignee: Assignee
-tickets.contract: Contract
+tickets.contract: Ongoing contract
```

Comment:
If you change the value of the key, it will be replaced wherever the key is used. That's not what we want, even if it's used only in one place.

Also, there is already a key `tickets.contracts.ongoing` which contains the expected string. So a better solution is to replace the key in the template (at the end of [`templates/tickets/show.html.twig`](https://github.com/Probesys/bileto/blob/main/templates/tickets/show.html.twig#L511)).